drivers/dma/dma_stm32.c

```diff
@@ -258,14 +225,14 @@ static int dma_stm32_configure(struct device *dev, u32_t id,
 	}
 
 #ifdef CONFIG_DMA_STM32_V1
-if ((stream->direction == MEMORY_TO_MEMORY) &&
+	if ((stream->direction == MEMORY_TO_MEMORY) &&
```

Comment:
Move this to previous commit

Reply:
this part is in 6498dd86cf7ecf6b55f55bd2f94b77defed7c97f